src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.h

```diff
@@ -57,6 +57,21 @@ class BtreeLBAMapping : public BtreeNodeMapping<laddr_t, paddr_t> {
 //
 // NOTE THAT, for direct BtreeLBAMappings, their intermediate_keys are the same as
 // their keys.
+//
+// Indirect BtreeLBAMapping can be further categrozied into HALF_INDIRECT and
+// FULL_INDIRECT.
+// 1. HALF_INDIRECT mappings: indirect mappings that don't have information from
+//    the intermediate mapping. Specifically, they don't have valid intermediate_base,
+//    intermediate_offset or intermediate_length.
```

Comment:
Similarly,

HALF_INDIRECT -> RAW/INCOMPLETE_INDIRECT
FULL_INDIRECT -> INDIRECT

1. RAW/INCOMPLETE_INDIRECT mappings: they don't have information from the intermediate mapping and should not be exposed outside LBAManager. Specifically, they don't have valid intermediate_base, intermediate_offset, intermediate_length or attached to a logical cached extent.
2. INDIRECT mappings: they have the complete information from intermediate mappings for LBAManager users.